.stylelintrc.js

```diff
@@ -2,85 +2,33 @@ module.exports = {
-        'no-extra-semicolons'                              : true,
         'no-invalid-double-slash-comments'                 : true,
-        'number-leading-zero'                              : 'always',
         'number-max-precision'                             : 3,
-        'number-no-trailing-zeros'                         : true,
-        'property-case'                                    : 'lower',
         'rule-empty-line-before'                           : ['always', { ignore: ['after-comment'], except: ['inside-block-and-after-rule', 'first-nested'] }],
-        'selector-attribute-brackets-space-inside'         : 'never',
-        'selector-attribute-operator-space-after'          : 'never',
-        'selector-attribute-operator-space-before'         : 'never',
         'selector-class-pattern'                           : null,
-        'selector-combinator-space-after'                  : 'always',
-        'selector-combinator-space-before'                 : 'always',
-        'selector-list-comma-newline-after'                : 'never-multi-line',
-        'selector-list-comma-newline-before'               : 'never-multi-line',
-        'selector-list-comma-space-after'                  : 'always',
-        'selector-list-comma-space-before'                 : 'never',
-        'selector-max-empty-lines'                         : 0,
-        'selector-pseudo-class-case'                       : 'lower',
         'selector-pseudo-class-no-unknown'                 : true,
-        'selector-pseudo-class-parentheses-space-inside'   : 'never',
-        'selector-pseudo-element-case'                     : 'lower',
         'selector-pseudo-element-colon-notation'           : 'single',
         'selector-pseudo-element-no-unknown'               : true,
         'selector-type-case'                               : 'lower',
         'selector-type-no-unknown'                         : [true, { ignoreTypes: ['from', 'to', '0%', '50%', '100%', '_'] }],
         'shorthand-property-no-redundant-values'           : true,
         'string-no-newline'                                : true,
-        'string-quotes'                                    : 'single',
         'time-min-milliseconds'                            : 100,
-        'unit-case'                                        : 'lower',
-        'unit-whitelist'                                   : ['px', 'em', 'rem', '%', 'vw', 'vh', 'deg', 'ms', 's', 'fr'],
+        'unit-allowed-list'                                : ['px', 'em', 'rem', '%', 'vw', 'vh', 'deg', 'ms', 's', 'fr'],
         'value-keyword-case'                               : 'lower',
-        'value-list-comma-newline-after'                   : 'never-multi-line',
-        'value-list-comma-newline-before'                  : 'never-multi-line',
-        'value-list-comma-space-after'                     : 'always',
-        'value-list-comma-space-before'                    : 'never',
```

Comment:
removed all the rules that are deprecated in stylelint v15.
please let me know if any concerns.

Reply:
doesnt it break the style lints?
maybe any major changes?

Reply:
there is no automated linting like prettifier on this project, and after removing these rules it looks good, nothing is broken, no errors. Unfortunately, there are no equivalents to these deprecated rules in stylelint 15.
They really look useless.
Shoul I leave them and silence deprecation warning instead?